doc/conf.py

```diff
@@ -170,6 +170,16 @@
 # Options for LaTeX output
 # ------------------------
 
+# Notes for customization
+# https://www.sphinx-doc.org/en/master/latex.html
+# https://github.com/sphinx-doc/sphinx/issues/4136
+
+# Use a latex engine allows for unicode characters in docstrings
+latex_engine = "xelatex"
+
+# Docs on xindy: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-latex_use_xindy
+latex_use_xindy = True
+
```

Comment:
```suggestion
# Use a latex engine allows for unicode characters in docstrings
latex_engine = "xelatex"
```

My vote would be to keep this simple. `xindy` is the default with xelatex so no need to explicitly specify it here. The links are more valuable from a troubleshooting perspective and won't be relevant (:crossed_fingers: )